docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -848,3 +848,37 @@ When I find = `5` elements by `By.xpath(//script):a` and for each element do
 |When I set 'type' attribute value of the context element to the 'scenario' variable 'type'|
 |Then `${type}` is equal to `text/javascript`                                              |
 ----
+
+=== Browser Configuration Steps
+==== Set the browser command line arguments
+
+Allows to set command line arguments for browsers.
+This step works similar to the property
+```properties
+web.driver.<browser-name>.command-line-arguments
+```
+In case both property and step are used - arguments from property will be ignored.
```

Comment:
Worth clarifying if the settings are going to be used for the **next** session or globally